FSRS v1 and v2 (#144)
open-spaced-repetition/fsrs-benchmark: README.md

### Algorithms

> - FSRS v1 and v2: alpha versions of FSRS that weren't publicly available.

Reviewer: Nope. They are public: https://www.reddit.com/r/Anki/comments/xhfl0p/new_progress_in_implementing_the_custom_algorithm/

Author: How would you phrase this then?

Author: "Weren't available to be used for custom scheduling in Anki" sounds awkward. @user1823 maybe you have a suggestion

Reviewer: They were available to be used for custom scheduling. I provided the script and optimizer: 

https://github.com/open-spaced-repetition/fsrs4anki/blob/v1.0.0-alpha/fsrs4anki_scheduler.js

https://github.com/open-spaced-repetition/fsrs4anki/blob/v1.0.0-alpha/fsrs4anki_optimizer.ipynb

Author: Then we need to change the description of FSRS v3, since it's not actually "the first version of the FSRS algorithm that people actually used."

Reviewer: FSRS v3 is the first version support FSRS helper add-on: https://www.reddit.com/r/Anki/comments/xw6jq3/big_update_in_fsrs4anki_v300/

> - FSRS v1 and v2: alpha versions of FSRS that weren't publicly available.

Reviewer 2: ```suggestion
- FSRS v1 and v2: the initial experimental versions of FSRS.
```

> - FSRS v3: the first version of the FSRS algorithm that people actually used.

Reviewer 2: ```suggestion
- FSRS v3: the first official release of the FSRS algorithm, made available as a custom scheduling script.
```
